Cargo.toml

```diff
@@ -16,7 +16,9 @@ serde = { version = "1.0.130", features = ["derive", "rc"] }
 thiserror = "2"
 url = "2.5.3"
 deno_error = "0.5.1"
-capacity_builder = "0.4.0"
+capacity_builder = { version = "0.5.0", features = ["ecow", "hipstr"] }
+hipstr = "0.6"
+ecow = { version = "0.2.3", features = ["serde"] }
```

Comment:
We can swap out these dependencies to try out other stuff. hipstr allows up to 23 bytes so we use that for package names (scoped names are mostly shorter than that) and then ecow is 15 bytes, so used for shorter stuff like tag names, pre/build parts.